recipes/swig/all/patches/0002-4.1.0-do-not-define-SWIG_LIB_WIN_UNIX.patch

```diff
@@ -0,0 +1,11 @@
+--- configure.ac
++++ configure.ac
+@@ -2825,7 +2825,7 @@ case $build in
+   *-*-cygwin*) SWIG_LIB_WIN_UNIX=`cygpath --mixed "$SWIG_LIB"`;;
+   *) SWIG_LIB_WIN_UNIX="";;
+ esac
+-AC_DEFINE_UNQUOTED(SWIG_LIB_WIN_UNIX, ["$SWIG_LIB_WIN_UNIX"], [Directory for SWIG system-independent libraries (Unix install on native Windows)])
++AC_DEFINE_UNQUOTED(SWIG_LIB_WIN_UNIX, [""], [Directory for SWIG system-independent libraries (Unix install on native Windows)])
+ 
+ SWIG_LIB_PREINST=$ABS_SRCDIR/Lib
+ AC_SUBST(SWIG_LIB_PREINST)
```

Comment:
Another thing worth mentioning is that as of 4.1.0, SWIG has a CMakeLists.txt available for building.

Reply:
YES! 🚀